docs/docs/quickstart.mdx

```diff
@@ -0,0 +1,82 @@
+---
+title: Quickstart
+hide_title: false
+sidebar_position: 2
+---
+
+**Ready to give Apache Superset a try?** This quickstart will help you run Superset on your local machine in **5 simple steps**.
+It assumes that you have [Docker](https://www.docker.com) installed.
+
+### 1. Get Superset
+To get started, add the latest Superset version in the `SUPERSET_VERSION` environment variable, and pull the container from Docker Hub:
+```
+$ export SUPERSET_VERSION="";
+         docker pull apache/superset:$SUPERSET_VERSION
+```
+:::tip
+Not sure what the latest stable Superset version is? [Click here](https://github.com/apache/superset/releases) to check our release page 
+for up-to-date information.
+:::
+
+### 2. Start Superset
+```
+$ docker run -d -p 8080:8088 \
+             -e "SUPERSET_SECRET_KEY=$(openssl rand -base64 42)" \
+             -e "TALISMAN_ENABLED=False" \
+             --name superset apache/superset:$SUPERSET_VERSION
+```
+:::tip
```

Comment:
Maybe this should be placed before the command so users set the config before running.

Reply:
You mean the TALISMAN environment variable, or about the container?

Reply:
I mean placing the tip before the command:

Note that some configuration is mandatory for Superset in order to start. In particular, Superset will not start without a user-specified value of `SECRET_KEY` in a Superset configuration file or `SUPERSET_SECRET_KEY` as an environment variable.Please see [Configuring Superset](https://superset.apache.org/docs/installation/configuring-superset/) for more details.

```
$ docker run -d -p 8080:8088 \
             -e "SUPERSET_SECRET_KEY=$(openssl rand -base64 42)" \
             -e "TALISMAN_ENABLED=False" \
             --name superset apache/superset:$SUPERSET_VERSION
```

Reply:
Ah, that can be done too, just made the change on commit https://github.com/apache/superset/pull/26129/commits/229bf700e80ab2d49d49b4eaecfeea8ee3415c20